practice/react-practice-one/src/components/Progress/Progress.jsx

```diff
@@ -0,0 +1,13 @@
+/* Import CSS */
+import './progress.css';
+
+const CircularProgress = () => {
+  return (
+    <div className="progress">
+      <div className="progress-loading"></div>
+      <span className="visually-hidden">Loading...</span>
```

Comment:
@JennyDoJD If another place needs to modify 'Loading...' to 'Processing...' or something else, how should we handle it?